src/common/types.tsx

```diff
@@ -2,6 +2,14 @@ import { PublicKey } from '@solana/web3.js';
 import { ReactNode, Dispatch, SetStateAction } from 'react';
 import { SetterOrUpdater } from 'recoil';
 
+export interface ItokenOwned {
+  mint: string;
+  amountHeld: number;
+  memberCount: number;
```

Comment:
Don't think the object to store token data is the right place to store the number of members of a community for a token.

Reply:
this is an object meant to be passed to a front end component

Reply:
fetched now